modules/ROOT/pages/logging/query-log-analyzer.adoc

```diff
@@ -0,0 +1,101 @@
+[[aura-monitoring]]
+= Query log analyzer
+
+label:AuraDB-Professional[]
+label:AuraDB-Enterprise[]
+
+Query log analyzer is a feature that provides a UI to review the queries executed on an Aura instance.
+
+You can access query logs from *Logs* in the left-hand navigation or from query-related metrics in the *Database* tab in *Metrics*.
+Use the more menu (*...*) on the metric to navigate to *Explore query logs*.
+
+To toggle between instances, use the dropdown menu on the top left.
+
+Query log analyzer is split up in three parts:
+
+* *Query timeline* - Timeline showing metrics for number of queries, failed queries and query latency.
+* *Summary table* - An aggregated view of query logs, giving a high level overview over the selected time period.
+* *Details table* - A detailed view showing individual query executions in the selected time period.
+
+To fetch logs, first select a time range in the Query timeline.
+With a time selection done, press the *Fetch logs* button.
+You may optionally select any filters or search text if required, then press *Fetch*.
+
+A summary of query executions is returned, showing aggregations per query.
+To see the individual query executions, click the right arrow at the end of the line to show details for that query.
+The details pane shows individual executions.
+
+== Timeline interactions
```

Comment:
```suggestion
== Query timeline
```

Reply:
This was changed in the latest revision of the document to reflect the exact name used in the UI

Reply:
Updated!